NuGet.config

```diff
@@ -2,6 +2,10 @@
 <configuration>
   <packageSources>
     <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <add key="darc-pub-dotnet-core-setup-7d57652" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-7d57652f/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-corefx-4ac4c03" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-4ac4c036/nuget/v3/index.json" />
+    <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
```

Comment:
Didn't see similar changes in the other 3.0 -> 3.1 merges. Is this necessary?

Reply:
No. Yeah, it can be removed.